### PR TITLE
OKTA-562221 Update SCIM reference docs with example

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added a `PUT` request example for [Update specific Group Membership](https://developer.okta.com/docs/reference/scim/scim-20/#update-specific-group-membership) section.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-562221](https://oktainc.atlassian.net/browse/OKTA-562221)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md [hunk 1]
```diff
@@ -697,29 +697,47 @@ To add or remove users inside a specific pushed Group object on the SCIM server,
 If these three requirements are met, Okta sends a request to add the specified users to the Group object on the SCIM server.
 
 * For all new OIN app integrations, this request to update a Group object is sent through a PATCH method request.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md:699]:
```suggestion
* For all new OIN app integrations, a PATCH method request is used to update a Group object. For example:
```

File: packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md [hunk 2]
```diff
@@ -697,29 +697,47 @@ To add or remove users inside a specific pushed Group object on the SCIM server,
...
+                "display": "test.user@okta.local"
+            }]
         }]
-    }]
-}
-```
+    }
+    ```
+
+* For custom app integrations created using the AIW, this request to update a Group object is sent through a PUT request.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md:723]:
```suggestion
* For custom app integrations created using the AIW, a PUT method request is used to update a Group object. For example:
```

File: packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md [hunk 3]
```diff
@@ -697,29 +697,47 @@ To add or remove users inside a specific pushed Group object on the SCIM server,
...
-          "op": "add",
-          "path": "members",
-          "value": [{
-              "value": "23a35c27-23d3-4c03-b4c5-6443c09e7173",
-              "display": "test.user@okta.local"
+    {
+        "schemas": ["urn:ietf:params:scim:api:messages:2.0:PatchOp"],
+        "Operations": [{
+            "op": "remove",
+            "path": "members[value eq \"89bb1940-b905-4575-9e7f-6f887cfb368e\"]"
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md:710]: @williamyang-okta do we need to explain that we are removing a member here, and adding a different member to the group in this example?

**Review by vanngo-okta — APPROVED** [review]
LGTM. Just a suggestion with the leading example sentence. Approving to not hold you up.

File: packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md (hunk 3 above)
Comment (williamyang-okta) [reply packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md:710]: I feel it's pretty clear, not needed.

**Review by williamyang-okta — APPROVED** [review]
LGTM

**Pull request merged**